engine/language-server/src/main/scala/org/enso/languageserver/runtime/ContextRegistryProtocol.scala

```diff
@@ -231,9 +231,15 @@ object ContextRegistryProtocol {
         )
       }
 
+      /** Indicates that an expression is pending a computation
+        */
       case class Pending(message: Option[String], progress: Option[Double])
           extends Payload
 
+      /** Indicates that an expression's computation has been interrupted and shall be retried.
+        */
+      case object PendingInterrupted extends Payload
```

Comment:
Shouldn't this be a `wasInterrupted: Boolean` flag on the `Pending` type instead?

I'm not sure why these need to be separate. In the future, won't we also want to report progress on computation that is retried after interruption?

Reply:
That sounds OK, I have no strong opinion about this.